src/providers/adobe.ts

```diff
@@ -0,0 +1,91 @@
+import { $fetch } from 'ofetch'
+import { hash } from 'ohash'
+
+import type { FontProvider, ResolveFontFacesOptions } from '../types'
+import { extractFontFaceData, addLocalFallbacks } from '../css/parse'
+import { cachedData } from '../cache'
+import { logger } from '../logger'
+
+interface ProviderOption {
+  id?: string
```

Comment:
in terms of your proposed next steps, should we perhaps also support an array of ids?